recipe/meta.yaml

```diff
@@ -25,8 +25,8 @@ requirements:
     # Note that "testing" in conda-forge should be limited to testing
     # successful packaging, and not testing code.
     - compilers
-    - clang_osx-64 =11.1.0=hb91bd55_1  # [osx and x86_64]
-    - clang_osx-arm64 =11.1.0=h40be2b1_1  # [osx and arm64]
+    - clang_osx-64 =11.1.0  # [osx and x86_64]
+    - clang_osx-arm64 =11.1.0  # [osx and arm64]
```

Comment:
Do you mean any version that is `11.1.0.*` or any `11.1.*`? I suspect the second one but maybe not?

Reply:
In this case, I believe I wanted to avoid any possibility of binary (ABI) incompatibility with the pre-compiled eups packages.

Reply:
(Admittedly, this could still be a concern with even the master branch.)